README.md

```diff
@@ -53,3 +54,11 @@ Feel free to raise and fix issues.
 For any questions, join #oshc-dev on Slack. Get an invite [here](https://opensourcehelp.herokuapp.com/).
 
 Note : All design related tasks have reward associated with them.
+
+
+<!-- develop -->
```

Comment:
I am not familiar with conflicts. But it is now resolved or should i run these command and check?